recipes/darglint/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "darglint" %}
+{% set version = "0.6.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "d644a3fbe6a80ae73863d79e42be3896447712394930d5e7156bf7476792a243"
```

Please remove the quotes as otherwise the autotick bot doesn't work.
```suggestion
  sha256: d644a3fbe6a80ae73863d79e42be3896447712394930d5e7156bf7476792a243
```